app/manifest.json

```diff
@@ -60,6 +60,7 @@
     "http://localhost:8545/",
     "https://*.infura.io/"
     ],
+  "unlimitedStorage": true,
```

Comment:
`unlimitedStorage` should be added to the permissions, the same as `storage`. Referenced: [Declare Permissions](https://developer.chrome.com/extensions/declare_permissions)